src/companies/factory/stock.py

```diff
@@ -4,14 +4,30 @@
 
 from app.testing import register
 from app.testing.factory import FixtureFactory
-from companies.models.stock import Material, Stock
+from companies.models import Material, MaterialType, Stock
 
 
 @register
 def material(self: FixtureFactory, **kwargs: dict) -> dict:
     return self.mixer.blend(Material, **kwargs)
 
 
+@register
+def material_type(self: FixtureFactory, **kwargs: dict) -> dict:
+    return self.mixer.blend(MaterialType, **kwargs)
+
+
+@register
+def material_type_data(self: FixtureFactory, **kwargs: dict) -> dict:
+    schema = self.schema(
+        schema=lambda: {
+            "name": self.field("name"),
```

Comment:
name refers to human name, word is a better option I think

```suggestion
            "name": self.field("word"),
```